src/Config/Dumper/DumperInteface.php

```diff
@@ -0,0 +1,28 @@
+<?php
+
+/**
+ * This file is part of Contao.
+ *
+ * Copyright (c) 2005-2015 Leo Feyer
+ *
+ * @license LGPL-3.0+
+ */
+
+namespace Contao\CoreBundle\Config\Dumper;
+
+/**
+ * DumperInteface is the interface implemented by internal cache dumpers.
+ *
+ * @author Andreas Schempp <https://github.com/aschempp>
+ */
+interface DumperInteface
```

Comment:
I suppose the file name, the interface name etc. should read "DumperInte**r**face", shouldn't it?